autopilot/autopilot.go

```diff
@@ -121,7 +120,7 @@ type Autopilot struct {
 }
 
 // New initializes an Autopilot.
-func New(cfg config.Autopilot, bus Bus, workers []Worker, logger *zap.Logger) (_ *Autopilot, err error) {
+func New(cfg config.Autopilot, bus Bus, workers []Worker, allowRedundantIps bool, logger *zap.Logger) (_ *Autopilot, err error) {
```

Comment:
setting should be in the `config.Autopilot`

Reply:
I considered that but that would expose the setting in our yaml file. I thought we would not want that...

Reply:
I believe Nate meant to move this from the API settings to the config rather than not exposing it at all. But in case I'm wrong we can probably add a `-` or something to the field to ignore it in the config.